Basic-Car-Maintenance/Documentation.docc/getting-started.tutorial

```diff
@@ -88,8 +88,56 @@
             }
         }
     }
-    
-    @Section(title: "Starting on an issue") {
+
+    @Section(title: "Setting Up Firebase Local Emulator") {
+        @ContentAndMedia {
+            Follow these steps to set up the Firebase Local Emulator to load data locally and not affect production. Please do not skip this step.
+        }
+
+        @Steps {
+            @Step {
+                Install Homebrew, a package manager for macOS, if you haven't already:
+
+                `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`
+            }
+            @Step {
+                Install Xcode command line tools:
+
+                `xcode-select --install`
+            }
+            @Step {
+                Install Node Version Manager (NVM):
+
+                `brew install nvm`
+
+                Installs node version manager, so you don't need to update the system node version.
+            }
```

Comment:
> Installs node version manager, so you don't need to update the system node version.

This step doesn't show because you can't have 3 "steps" and a warning shows this in Xcode